package com.java.Day9.ExceptionHandling;

public class DividebyZero {

	public static void main(String[] args) {
		int no1 = 5, no2 = 0;
		System.out.println("Division: " +(no1/no2));
		//example of runtime exception
	}

}
